src/components/views/rooms/RoomSettings.js

```diff
@@ -321,18 +321,18 @@ module.exports = React.createClass({
 
         var state_default = (parseInt(power_levels ? power_levels.state_default : 0) || 0);
 
-        var room_aliases_level = state_default;
-        if (events_levels['m.room.aliases'] !== undefined) {
-            room_aliases_level = events_levels['m.room.aliases'];
```

Comment:
I moved this out to the js sdk in https://github.com/matrix-org/matrix-react-sdk/pull/187 / https://github.com/matrix-org/matrix-js-sdk/pull/94 so maybe we should just merge that and use it.